homeassistant/components/input_boolean/strings.json

```diff
@@ -17,5 +17,23 @@
         }
       }
     }
+  },
+  "services": {
+    "toggle": {
+      "name": "Toggle",
+      "description": "Toggles the helper on/off."
+    },
+    "turn_off": {
+      "name": "Turn off",
+      "description": "Turns off the helper."
+    },
+    "turn_on": {
+      "name": "Turn on",
+      "description": "Turns on the helper."
+    },
+    "reload": {
+      "name": "Reload",
+      "description": "Reloads helpers from the YAML-configuration."
```

Comment:
Isn't this a bit too generic as text?
Also it doesn't reload all helpers right, just this one?

Reply:
When used, in the UI, there is more context. You can't look at a single part of a translation string on its own.